migration/sql-files/034-token-resource-scopes.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE token_resource ADD scopes VARCHAR;
+ALTER TABLE token_resource ADD status INTEGER;
```

Comment:
You would be adding the `CREATE` SQLs too, right?

Reply:
They were created a long time ago :)

https://github.com/fabric8-services/fabric8-auth/blob/master/migration/sql-files/019-authorization-part-2.sql